helpers.go

```diff
@@ -0,0 +1,17 @@
+package substrate
```

Comment:
can you change the file name to utils_test.go otherwise this code now will be part of the library build not only the test suit.

By making the file a test file it will not be included in the final build and will be instead available only during the tests

Reply:
I got it, I'll fix this now